### ISSUE TITLE
Latest Tomcat Version has regressions - so please update to newer fixed version when they become available.

### ISSUE THREAD

**Issue by logopk** [docker-library/tomcat#306]
Note: There are known regressions: with jdbc-pool (see bug [67664](https://bz.apache.org/bugzilla/show_bug.cgi?id=67664)) and with Connector configurations when compression is enabled (see bug [67670](https://bz.apache.org/bugzilla/show_bug.cgi?id=67670)). They will be fixed in the next release.

As eg 9.0.81 is not yet available as image, please make 9.0.82 available as soon as possible.

**Comment by cmartineauOptimum** [comment]
This broke our build as well

**Comment by yosifkit** [comment]
`9.0.81` was pushed just yesterday with the merge of https://github.com/docker-library/official-images/pull/15438 and `9.0.82` will be built/pushed once https://github.com/docker-library/official-images/pull/15538 is merged.

**Comment by jgournet** [comment]
Hi @yosifkit ,
latest image of 8-jdk8-corretto seems to be affected by the compression issue.
Is there a way to build&push a new version of it please ?

**Comment by yosifkit** [comment]
See this repo's readme for info about how a change merged here get to Docker Hub: https://github.com/docker-library/tomcat/tree/40654f4c9befbaf31ae774c82c76e0a7f0b0c2fc#see-a-change-merged-here-that-doesnt-show-up-on-docker-hub-yet.

tl;dr: `tomcat:8*` images will be updated after https://github.com/docker-library/official-images/pull/15554 is merged.
